statsmodels/regression/linear_model.py

```diff
@@ -818,21 +822,42 @@ def iterative_fit(self, maxiter=3):
         ----------
         maxiter : integer, optional
             the number of iterations
+        rtol : float, optional
+            Relative tolerance between estimated coefficients to stop the
+            estimation.  Stops if
+
+            max(abs(last - current) / abs(last)) < rtol
+
         """
-        #TODO: update this after going through example.
-        for i in range(maxiter-1):
+        # TODO: update this after going through example.
+        for i in range(maxiter - 1):
             if hasattr(self, 'pinv_wexog'):
                 del self.pinv_wexog
             self.initialize()
             results = self.fit()
+            self.history.append(results.params)
+            if i == 0:
+                last = results.params
+            else:
+                diff = np.max(np.abs(last - results.params) / np.abs(last))
```

Comment:
This will not stop early if some params become very small, being close to zero.
I think adding also atol and using np.allclose with both would be better.